lightning-invoice/src/utils.rs

```diff
@@ -228,7 +230,8 @@ where
 	let duration = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH)
 		.expect("for the foreseeable future this shouldn't happen");
 	create_invoice_from_channelmanager_and_duration_since_epoch(
-		channelmanager, keys_manager, network, amt_msat, description, duration
+		channelmanager, keys_manager, network, amt_msat,
+		description, duration, invoice_expiry_delta_secs
```

Comment:
nit: would just wrap at 100 chars instead of making them evenly distributed to ease future automation